README.md

```diff
@@ -56,6 +57,69 @@ Add new records to your mongodb instance by HTTP PUT using the following route:
 * `http://localhost:3000/scicat/Publication`
 
 
+## ENVIRONMENT Variables
+
+*System variables**
+ Key | Description | Default
+ --------:|-------------| --------
+DAPP_ID | No Idea | oai-pmh-service
+CONNECTOR | (don't change) | mongodb
+ADMIN_USER_EMAIL | E-Mail address of the admin user | none
+LOG_LEVEL | default/error/warning | none  
```

Comment:
default should probably be `error`, but that can be done in an other change

Reply:
Fixed with [95b512b](https://github.com/SciCatProject/oai-provider-service/pull/112/commits/95b512b51d755c65dad1ee0d12fbb0dfc953a9d8)